framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -65,27 +61,13 @@ SideSetsAroundSubdomain::SideSetsAroundSubdomain(const InputParameters & paramet
 }
 
 void
-SideSetsAroundSubdomain::initialize()
-{
-  // Initialize the BlockRestrictable parent
-  initializeBlockRestrictable(_pars);
-}
-
-void
 SideSetsAroundSubdomain::modify()
 {
   // Reference the the libMesh::MeshBase
   MeshBase & mesh = _mesh_ptr->getMesh();
 
-  // Extract the 'first' block ID
-  SubdomainID block_id = *blockIDs().begin();
-
-  // Extract the SubdomainID
-  if (numBlocks() > 1)
-    mooseWarning("SideSetsAroundSubdomain only acts on a single subdomain, but multiple were "
-                 "provided: only the ",
-                 block_id,
-                 "' subdomain is being used.");
+  // Extract the block ID
+  SubdomainID block_id = (_mesh_ptr->getSubdomainIDs({getParam<SubdomainName>("block")}))[0];
```

Comment:
Oh, the SubdomainName is now just a single parameter - it's not a vector of Names.  So the warning message is irrelevant.

I don't understand why this MeshModifier originally derived from ```BlockRestrictable```.  No others do, so that's why i was a bit more confident about removing it.  I wonder whether it was just a historical quick-fix way to have a "block = ..." in the input file.  Anyway, clearly ```BlockRestrictable``` doesn't play nicely with adding blocks if you use it naively.

I'd like to merge this PR.  I know what "busy for the next several weeks" means!